src/toolkit/enrichment.h

```diff
@@ -31,7 +31,7 @@ class Assays {
 };
 
 /// @param mat the material inquired about
-/// @return the atom percent of U-235 w.r.t U-235+U-238 in a material
+/// @return the mass percent of U-235 w.r.t U-235+U-238 in a material
```

Comment:
For stability reasons, I don't think that you can do this.  I think that you need to add a different state var (UraniumAssayByMass?) and then make them both optional.  This change will break existing input files.